src/main/java/com/fasterxml/jackson/databind/introspect/AnnotatedCreatorCollector.java

```diff
@@ -274,6 +274,14 @@ Creators collect(JavaType type, Class<?> primaryMixIn)
         return result;
     }
 
+    private boolean _isIncludableFactoryMethod(Method m)
```

Comment:
This can be static, as well as the similar method in AnnotatedMethodCollector. I'll change both in a stacked commit in this PR.